tests/test_multidict_.py

```diff
@@ -0,0 +1,133 @@
+import pytest
```

Comment:
I understand that this might look not needed, but it'd be great to see docstrings around :)
pytest picks those up from test functions and module docstring is a good teaser of what's going to be tested in here in general.
Although it's a nitpick, I'm pretty sure that tests are no exception.

Reply:
Docstrings in tests?
Looks like a sort of overengeneering, especially for such slow evolved and limited library as multidict.
Many people *use* multidict directly or indirectly but how many send patches?

Reply:
It's more about being clean and caring about future self + others, who might be not sending patches also because it's hard or takes a lot of time to understand.

Reply:
Honestly, I don't care.
@webknjaz if you want to add docstrings everywhere in test suite -- you are welcome!

Reply:
I want people to do this when contributing new things at least. Going over the existing code base is more resource consuming...

Reply:
@gyermolenko consider https://youtu.be/o9pEzgHorH0